apps/android/LiveVideoApp/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Oh interesting another .gitignore. So there seems to be some inconsistency. The main .gitignore references sub directories such as `sdk/libs/`. So why wouldn't it also reference `.idea/shelf/` for example? I don't understand why there is inconsistent strategies.

Reply:
the .gitignore in the .idea folder is created by android studio and can be ignored.

Reply:
Yes, that's right the second .gitignore is auto generated by Android Studio, but yeah, it can be ignored Or I just can remove it.

Reply:
But what about the 3rd .gitignore under the `apps` directory? Is that similar?

Reply:
We talked offline and this .gitignore is also generated automatically. So I'm good with leaving it in.